app/views/events/show.html.erb

```diff
@@ -1,3 +1,4 @@
+<% provide :title, @event.organizer_name %>
```

Comment:
We should state @event.name here instead of @event.organizer_name as this event view is used for organizers and users alike. :)